.travis.yml

```diff
@@ -14,7 +14,7 @@ before_script:
 script:
     - DCO_VERBOSITY=-q ../project/script/validate/dco
     - ../project/script/validate/fileheader ../project/
-    - go test -v -race -covermode=atomic -coverprofile=coverage.txt ./...
+    - sudo $GOROOT/bin/go test -test.root -test.v -race -covermode=atomic -coverprofile=coverage.txt ./...
```

Comment:
Tried to use `export PATH=$GOROOT/bin:$PATH`, seems not quite work out.

Reply:
```
travis@travis-job-f1c1eeb9-d213-4c5b-bd66-aa15232af65a:~/gopath/src/github.com/meetup/aufs-test$ export PATH=$GOROOT/bin:$PATH
travis@travis-job-f1c1eeb9-d213-4c5b-bd66-aa15232af65a:~/gopath/src/github.com/meetup/aufs-test$ sudo go test -v -race -covermode=atomic -coverprofile=coverage.txt ./...
sudo: go: command not found
```

Reply:
Why not simply use `go test -c` as in former revisions?

Reply:
I thought we have `sudo: go: command not found` issue?

Reply:
So previous version didn't have that issue because we don't need `sudo go` for `go test -c`

Reply:
ping

Reply:
png @chenrui333